orc8r/cloud/deploy/terraform/orc8r-helm-aws/variables.tf

```diff
@@ -452,3 +458,26 @@ variable "cloudwatch_exporter_enabled" {
   default     = false
   type        = bool
 }
+
+
+##############################################################################
+# Domain proxy variables
+##############################################################################
+
+variable "dp_enabled" {
+  description = "Enable domain proxy"
+  type        = bool
+  default     = false
+}
+
+variable "dp_sas_endpoint_url" {
+  description = "Sas endpoint url where to connect DP to."
+  type        = string
+  default     = "https://fake-sas-service/v1.2"
+}
```

Comment:
Nit: don't use "fake" defaults -- "default" should mean "this will work if you don't touch it". Instead, don't put a default, but can add an "example =" comment

Reply:
fixed. I removed fake value.